tests/test_target.py

```diff
@@ -37,23 +36,57 @@ def build_targets():
 
     targets = []
 
+    if target_configs.get('AndroidTarget') is not None:
+        print('> Android targets:')
+        for entry in target_configs['AndroidTarget'].values():
+            pp(entry)
+            a_target = AndroidTarget(
+                connection_settings=entry['connection_settings'],
+                conn_cls=lambda **kwargs: AdbConnection(adb_as_root=True, **kwargs),
+            )
+            targets.append((a_target, None))
+
+    if target_configs.get('LinuxTarget') is not None:
+        print('> Linux targets:')
+        for entry in target_configs['LinuxTarget'].values():
+            pp(entry)
+            l_target = LinuxTarget(connection_settings=entry['connection_settings'])
+            targets.append((l_target, None))
+
     if target_configs.get('LocalLinuxTarget') is not None:
         print('> LocalLinux targets:')
         for entry in target_configs['LocalLinuxTarget'].values():
             pp(entry)
             ll_target = LocalLinuxTarget(connection_settings=entry['connection_settings'])
-            targets.append(ll_target)
+            targets.append((ll_target, None))
+
+    if target_configs.get('QEMUTargetRunner') is not None:
+        print('> QEMU target runners:')
+        for entry in target_configs['QEMUTargetRunner'].values():
+            pp(entry)
+            qemu_settings = entry.get('qemu_settings') and entry['qemu_settings']
```

Comment:
the `and ...` part is useless, `dict.get()` already supports providing a default value if the key does not exist (that's the only reason this function exists), and that default defaults to `None`. So you'll get the exact same result by dropping the `and ...`

Reply:
My bad. Forgot `dict.get()` here.